src/Microsoft.VisualStudio.Editors/PropPages/CSharpLanguageVersion.vb

```diff
@@ -8,6 +8,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
     Friend Class CSharpLanguageVersion
 
         Private Const s_languageVersion_Default As String = "default"
+        Private Const s_languageVersion__DisplayNameForDefault As String = "C# latest major version (default)"
```

Comment:
@OmarTawfik Follow [this](https://github.com/nguerrera/xliff-converter#convert-to-xlf-and-back)